keyboards/annepro2/annepro2.c

```diff
@@ -104,8 +104,8 @@ void keyboard_post_init_kb(void) {
     ap2_led_get_status();
 
     #ifdef RGB_MATRIX_ENABLE
-    ap2_led_enable();
```

Comment:
This is the line that is causing flicker on my keyboard. It enables lighting profiles first followed by the call to enable manual control. It is possible to prevent flicker by defaulting to manual control enabled in [Shine](https://github.com/OpenAnnePro/AnnePro2-Shine/blob/058af8537df266d8c539fdb10ae320f74c195e01/source/settings.c#L30) but I believe that the adjustment should be done here.

Another issue that is fixed by this PR - if I disable LEDs, disconnect the keyboard then it will light up momentarily before shutting down LEDs during the next connection.